Makefile

```diff
@@ -0,0 +1,17 @@
+box.phar:
+	curl -Lls https://github.com/humbug/box/releases/download/3.8.4/box.phar -o box.phar
```

Comment:
is it well understood here that if you change the link it is not enough to invalidate the existing `box.phar`?